pkg/noderesourcetopology/README.md

```diff
@@ -45,6 +45,9 @@ profiles:
         - default
         - production
         - test-namespace
+      excludelist:
```

Comment:
Can we describe the resources we need instead of excluding what we don't need？
Or if there is reason？

Reply:
By default, the Filter plugin should check NUMA-alignment for all resources, unless the user wants to exclude a specific resource from that check.